src/lib/eZ/RichText/Resources/stylesheets/xhtml5/edit/docbook.xsl

```diff
@@ -315,7 +315,25 @@
         </xsl:if>
       </xsl:if>
       <xsl:call-template name="ezattribute"/>
-      <xsl:apply-templates/>
+      <xsl:choose>
+        <xsl:when test="descendant::ezxhtml5:br">
+          <literallayout class="normal">
+            <xsl:for-each select="node()">
+              <xsl:choose>
+                <xsl:when test="local-name( current() ) = 'br'">
+                  <xsl:text>&#xA;</xsl:text>
+                </xsl:when>
+                <xsl:otherwise>
+                  <xsl:apply-templates select="current()"/>
+                </xsl:otherwise>
+              </xsl:choose>
+            </xsl:for-each>
+          </literallayout>
+        </xsl:when>
+        <xsl:otherwise>
+          <xsl:apply-templates/>
+        </xsl:otherwise>
+      </xsl:choose>
```

Comment:
wasn't there some "macro" a.k.a template for this already? Feels like it's a repetition.

Reply:
Fixed in https://github.com/ezsystems/ezplatform-richtext/pull/141/commits/07fa7018c70bab79f11522b28e010cd12f845c43. After investigating potential cleanup scenarios and internal sync with @alongosz we agreed that adding another `breaklineWithLiterallayout` template is a way to go.